Sources/Charcoal/Filters/Root/RootFilterViewController.swift

```diff
@@ -28,6 +28,7 @@ final class RootFilterViewController: FilterViewController {
         didSet { freeTextFilterViewController?.filterDataSource = freeTextFilterDataSource }
     }
 
+    var isFreeTextOnlySelected: Bool = false
```

Comment:
I think the naming of this var can be confusing: `isFreeTextOnlySelected`.
It sounds like setting it will enable the free text filter only. But that only happens in case you also shortcut to this filter at the same time, not in other contexts. If that makes sense?
What setting this var really does, is to make `FreeTextFilterViewController` dismiss filters on the next selection.

Perhaps we can rename it to something like that?
`dismissFiltersOnNextFreeTextSelection`? Or other, if you have a better name 😄

Reply:
Agree, I'll update the name with the one you suggest :D